paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/ResetConsumerAction.java

```diff
@@ -49,6 +51,17 @@ public ResetConsumerAction withNextSnapshotIds(Long nextSnapshotId) {
     @Override
     public void run() throws Exception {
         FileStoreTable dataTable = (FileStoreTable) table;
+        SnapshotManager snapshotManager = dataTable.snapshotManager();
+        Long latestSnapshotId = snapshotManager.latestSnapshotId();
+        if (nextSnapshotId != null
```

Comment:
Add a public static method, Otherwise, Otherwise, this check code would appear in three different places.